release-notes/opensearch-release-notes-2.19.3.md

```diff
@@ -0,0 +1,219 @@
+# OpenSearch and OpenSearch Dashboards 2.19.3 Release Notes
+
+## Release Details
+
+[OpenSearch and OpenSearch Dashboards 2.19.3](https://opensearch.org/versions/opensearch-2-19-3.html) includes the following bug fixes, infrastructure, documentation and maintenance updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/2.19/release-notes/opensearch.release-notes-2.19.3.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.19/release-notes/opensearch-dashboards.release-notes-2.19.3.md).
+
+
+## ENHANCEMENTS
+
+
+### Opensearch Flow Framework
+
+
+* Fix ApiSpecFetcher Memory Issues and Exception Handling ([#1192](https://github.com/opensearch-project/flow-framework/pull/1192))
+
+
+* Better handling of Workflow Steps with Bad Request status ([#1191](https://github.com/opensearch-project/flow-framework/pull/1191))
+
+
+## BUG FIXES
+
+
+### Opensearch Alerting
+
+
+* Backports #1850, #1854, #1856 to 2.19 ([#1858](https://github.com/opensearch-project/common-utils/pull/1858))
+
+
+### Opensearch Query Insights
+
+
+* Asynchronous search operations in reader ([#363](https://github.com/opensearch-project/query-insights/pull/363))
+
+
+### Opensearch Query Insights Dashboards
+
+
+* [BUG FIX] Enable Correct Sorting for Metrics in Query Insights Dashboard + dependent PRs ([#201](https://github.com/opensearch-project/query-insights-dashboards/pull/201))
+* [Bug Fix] Window size changing unexpectedly ([#203](https://github.com/opensearch-project/query-insights-dashboards/pull/203))
+* [Fix] Ensure accurate time filtering for Top Queries in OpenSearch 2.19+ ([#249](https://github.com/opensearch-project/query-insights-dashboards/pull/249))
+* Ensure accurate time filtering for Top Queries in OpenSearch 2.19 + revert [#219] ([#274](https://github.com/opensearch-project/query-insights-dashboards/pull/274))
+* Search bar fix ([#277](https://github.com/opensearch-project/query-insights-dashboards/pull/277))
+* [Fix] Explicitly match query by id and fix q scope in retrieveQueryById ([#269](https://github.com/opensearch-project/query-insights-dashboards/pull/269))
+
+
+### Opensearch Remote Metadata Sdk
+
+
+* Make generated responses robust to URL encoded id and index values ([#156](https://github.com/opensearch-project/opensearch-remote-metadata-sdk/pull/156))
+
+
+* Validate request fields in DDB Put and Update implementations ([#157](https://github.com/opensearch-project/opensearch-remote-metadata-sdk/pull/157))
+* Properly handle remote client search failures with status codes ([#158](https://github.com/opensearch-project/opensearch-remote-metadata-sdk/pull/158))
+
+
+### SQL
+
+
+* Fix: Date field format parsing for legacy query engine ([#3160](https://github.com/opensearch-project/sql/pull/3160))
+
+
+## INFRASTRUCTURE
+
+
+### SQL
+
+
+* Increment version to 2.19.3-SNAPSHOT ([#3601](https://github.com/opensearch-project/sql/pull/3601))
```

Comment:
Is this required? We might need to add for all components right?

Reply:
@Divyaasm Lets remove it form here